client/post-editor/controller.js

```diff
@@ -91,13 +92,21 @@ function maybeRedirect( context, postType, site ) {
 	return false;
 }
 
-function getPressThisContent( text, url, title ) {
-	return ReactDomServer.renderToStaticMarkup(
-		<p>
-			{ text ? <blockquote>{ text }</blockquote> : null }
-			via <a href={ url }>{ title }</a>.
-		</p>
-	);
+function getPressThisContent( text, url, title, image, embed ) {
+	let pieces = [];
```

Comment:
> const

My personal preference is that even if `const` lets me `push`, I still like to see `let` as an indicator that the value will change over time.